buildpipeline/pipeline.json

```diff
@@ -150,7 +150,8 @@
         "TreatWarningsAsErrors": "false"
       },
       "BuildParameters": {
-        "PB_ConfigurationGroup": "Release"
+        "PB_ConfigurationGroup": "Release",
+        "PB_BuildArguments": "-stripSymbols"
```

Comment:
Thanks for the explanation @dagood. The change LGTM. 

> Would it help to add purpose/result details to build-native.sh#L18?

What do you have in mind?